addons/flags/functions/fnc_handleScrollWheel.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: Timi007
+ * Handles the flag object height.
+ *
+ * Arguments:
+ * 0: Scroll amount <NUMBER>
+ *
+ * Return Value:
+ * Handled <BOOLEAN>
+ *
+ * Example:
+ * [5] call ace_flags_fnc_handleScrollWheel
+ *
+ * Public: No
+ */
+
+params ["_scrollAmount"];
+
+if (GVAR(isPlacing) isNotEqualTo PLACE_WAITING) exitWith {
+    false
+};
+
+// Move object height 10cm per scroll
+GVAR(objectHeight) = GVAR(objectHeight) + (_scrollAmount * 0.1);
+
+if (GVAR(objectHeight) < MIN_HEIGHT) then {
```

Comment:
`GVAR(objectHeight) = (MIN_HEIGHT max (GVAR(objectHeight) min MAX_HEIGHT));`

could get rid of the if statements and make this quite a bit faster, not sure if its easier to read though

Reply:
I could also use [BIS_fnc_clamp](https://community.bistudio.com/wiki/BIS_fnc_clamp) for more readability but will probably have worse performance because of a function call.